storage/cloud-client/notification_polling.py

```diff
@@ -64,6 +64,8 @@ def summarize(message):
     bucket_id = attributes['bucketId']
     object_id = attributes['objectId']
     generation = attributes['objectGeneration']
+    overwroteGeneration = attributes['overwroteGeneration']
```

Comment:
our style guide is `snake_case` for variable names, even if you're plucking them from the API repr.

Reply:
Oh dangit, I do that every time. I just can't stop Javaing.

Reply:
Hahaha. :)